components/styled/input/src/atoms/toggle.tsx

```diff
@@ -13,33 +13,38 @@ const StyledInput = styled.input`
   cursor: pointer;
   height: 1rem;
   width: 2rem;
-  background: linear-gradient(to left, rgba(0, 0, 0, 0) 50%, black 50%) right;
-  background-size: 200% 100%;
+  background: white;

```

Comment:
Personally I'm happy leaving this as is, but it's worth asking the question whether we want to make them style variables. In Banner > src > Index for example we're setting some. As I say I think this is probably overkill for just black/white...